DB/src/main/java/io/deephaven/db/v2/sources/DateTimeArraySource.java

```diff
@@ -104,17 +104,17 @@ protected void fillSparsePrevChunkUnordered(@NotNull final WritableChunk<? super
 
     @Override
     public void fillFromChunkByRanges(@NotNull OrderedKeys orderedKeys, Chunk<? extends Values> src) {
-        super.fillFromChunkByRanges(orderedKeys, src, DBTimeUtils::nanos);
+        super.<DBDateTime>fillFromChunkByRanges(orderedKeys, src, DBTimeUtils::nanos);
```

Comment:
None of these changes seem necessary. IntelliJ warns me: `Explicit type arguments can be inferred ` when I replicate your change locally.

Reply:
There are compile errors when removing this, since there is now a `DBTimeUtils.nanos(Instant instant)`.